Datastore/src/Connection/Rest.php

```diff
@@ -76,7 +77,8 @@ public function allocateIds(array $args)
      */
     public function beginTransaction(array $args)
     {
-        return $this->send('projects', 'beginTransaction', $args);
+        $argsWithReadTime = $this->setReadTime($args);
```

Comment:
Should this logic be in the client itself? At first look, it doesn't seem obvious to me this would also be handled in the GRPC connection layer.

Reply:
Timestamp for GRPC expects in epoch(Seconds) and REST expects in UTC DateTime format. Thats where this conversion is only required in the Rest.php

So, to have a uniform user experience in both REST and GRPC, We thought let the user send the epoch always and the REST converts when required.

Would this answer your question?